sdk/clientcore/http-stress/src/main/java/io/clientcore/http/stress/util/TelemetryHelper.java

```diff
@@ -100,7 +99,7 @@ public static void init() {
         AutoConfiguredOpenTelemetrySdkBuilder sdkBuilder = AutoConfiguredOpenTelemetrySdk.builder();
         String applicationInsightsConnectionString = System.getenv("APPLICATIONINSIGHTS_CONNECTION_STRING");
         if (applicationInsightsConnectionString != null) {
-            new AzureMonitorExporterBuilder().connectionString(applicationInsightsConnectionString).install(sdkBuilder);
+//            new AzureMonitorExporterBuilder().connectionString(applicationInsightsConnectionString).install(sdkBuilder);
```

Comment:
Following the changes from this [PR](https://github.com/Azure/azure-sdk-for-java/commit/c547d8f9aabe7e74d8266c19de18659f8fa587ca), we need to update this to 
```suggestion
 AzureMonitorExporter.customize(sdkBuilder, applicationInsightsConnectionString);
```